README.md

```diff
@@ -178,6 +179,15 @@ allow both IT and citizen developers to construct process-centric and case-centr
 - [TIBCO Cloud Integration](https://www.tibco.com/products/cloud-integration) - Empower everyone to integrate anything with API-led and event-driven integration.
 - [Workato](https://www.workato.com/) - A single platform for integration & workflow automation across your organization.
 
+### eiPaaS
+*Embedded Integration Platform as a Service.*
+- [apideck](https://www.apideck.com/) - A set of unified APIs to build native integrations with other SaaS providers. Available as REST and GraphQL APIs.
+- [Blendbase](https://github.com/blendbase/blendbase): Single open-source GraphQL API to connect CRMs to your SaaS. Query any customer CRM system (Salesforce, Hubspot and more) with a single API query from your SaaS app.
+- [Merge](https://www.merge.dev/) – One API to integrate with all HR, payroll, recruiting, accounting, ticketing, and CRM platforms.
```

Comment:
As regards to this solution, it is in line with [contribution guidelines](https://github.com/stn1slv/awesome-integration/blob/main/CONTRIBUTING.md#contributing):
>This means that the suggested software is:
> - widely recommended, regardless of personal opinion
> - highly discussed in the community due to its innovative nature
> - a niche product that fills a gap
> 

Probably, it would be better to replace it by some other solution that complies the guidelines (for example, Tray.io, Cyclr, Appmixer)